src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -46,42 +47,26 @@ public static synchronized void init() throws Exception {
     }
 
     private static synchronized void setUpMessageBus() throws Exception {
-        LOGGER.debug("Debug:setting up message buss");
+        LOGGER.debug("Debug:setting up message bus");
 
-        LOGGER.debug("before setting up message buss: amqpBroker: " + amqpBroker + ", conn: " + conn + ",cf:" + cf);
-        if (amqpBroker != null || conn != null || cf != null) {
+        LOGGER.info("Before setting up message bus, amqp broker: " + amqpBroker + ", connection: " + connection + ",connection factory:"
```

Comment:
Please change the log level of this to debug and remove the first log message in the method. Then you only get one at the beginning of the method, and one in the end - both on debug level.

Reply:
done